src/app/pages/system/CloudCredentials/cloudcredentials-form/cloudcredentials-form.component.ts

```diff
@@ -62,7 +68,9 @@ export class CloudCredentialsFormComponent {
     {
       type: 'input',
       name: 'client_secret-AMAZON_CLOUD_DRIVE',
-      placeholder: T('Application Key'),
+      placeholder: T('Application Client Secret'),
+      tooltip: T('Client secret for an existing Amazon Drive account.\
+                  This must come from an existing Amazon Drive account.'),
```

Comment:
By "existing", does this mean the account must have been created before the user can use it here?